tests/test_main.py

```diff
@@ -110,6 +110,17 @@ def test_main():
         "--num_test_tasks", "3", "--predicate_mlp_classifier_max_itr", "100"
     ]
     main()
+    # Cover the case where online learning stops after collecting max number of
+    # transitions.
+    sys.argv = [
```

Comment:
how long does this test take to run?

Reply:
on my computer, `test_main` went from 6.4s to 12.6s :/

Reply:
why not just lump this in with the test on the previous line?